e2e/nextjs/app-router/app/.editorconfig

```diff
@@ -0,0 +1,10 @@
+root = true
```

Comment:
Is this needed? or should it be ignored?

Reply:
normally it would be included, its some kind of editor config standard that next generates. But I dont think we need it in here